recipes-tag/xrt/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "xrt" %}
+{% set version = "1.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kklmn/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1c4e4db630298b9af2d98bef85f8c6c13a8dbac7ad707a8ea7f680b64a0c91d2
+ 
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - sphinx
+    - pyqt
```

Comment:
```suggestion
    - pyqt==5.6
```

Reply:
Unfortunately, we use the following versions on the floor:
- `pyqt=5.9.2=py36h22d08a2_1`
- `qt=5.9.6=h8703b6f_2`
Not sure if my suggested fix will work.

Reply:
I don't think it's needed anymore.